cdk/lib/payment-api.ts

```diff
@@ -1,22 +1,409 @@
+
+    const playApp = new GuPlayApp(this, {
+      app: "payment-api",
+      access: { scope: AccessScope.PUBLIC },
+      certificateProps: {
+        domainName: props.domainName,
+        hostedZoneId: "Z1E4V12LQGXFEC",
       },
+      monitoringConfiguration: { noMonitoring: true },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.SMALL),
+      scaling: props.scaling,
+      userData,
+      roleConfiguration: {
+        additionalPolicies: [
+          new GuAllowPolicy(this, "CloudwatchLogs", {
+            actions: [
+              "logs:CreateLogGroup",
+              "logs:CreateLogStream",
+              "logs:PutLogEvents",
+            ],
+            // TODO: Should we use a more specific resource here?
+            resources: ["arn:aws:logs:*:*:*"],
+          }),
+          new GuAllowPolicy(this, "SSMConfigParams", {
+            actions: ["ssm:GetParametersByPath"],
+            resources: [
+              `arn:aws:ssm:${this.region}:${this.account}:parameter/${app}/*`,
+            ],
+          }),
+          new GuAllowPolicy(this, "SqsMessages", {
+            actions: ["sqs:GetQueueUrl", "sqs:SendMessage"],
+            resources:
+              this.stage === "PROD"
+                ? [emailSqsCodeArn.valueAsString, emailSqsProdArn.valueAsString]
+                : [emailSqsCodeArn.valueAsString],
+          }),
+          new GuPutCloudwatchMetricsPolicy(this),
+          new GuAllowPolicy(this, "AssumeOphanRole", {
+            actions: ["sts:AssumeRole"],
+            resources: [ophanRole.valueAsString],
+          }),
+          new GuAllowPolicy(this, "KinesisPut", {
+            actions: ["kinesis:*"],
+            resources: [kinesisStreamArn.valueAsString],
+          }),
+          new GuAllowPolicy(this, "SQSPut", {
+            actions: ["sqs:SendMessage"],
+            resources:
+              this.stage === "PROD"
+                ? [
+                    contributionsStoreSqsQueueCodeArn.valueAsString,
+                    contributionsStoreSqsQueueProdArn.valueAsString,
+                  ]
+                : [contributionsStoreSqsQueueCodeArn.valueAsString],
+          }),
+          new GuAllowPolicy(this, "SqsKmsEncryption", {
+            actions: ["kms:Encrypt"],
+            resources: [sqsKmsArn.valueAsString],
+          }),
+          new GuGetS3ObjectsPolicy(this, "SettingsBucket", {
+            bucketName: "support-admin-console",
+            paths: this.stage === "PROD" ? ["*/*"] : ["CODE/*"],
+          }),
+        ],
+      },
+    });
+
+    // TODO: remove this tag after the migration
+    const playAppAsg = playApp.autoScalingGroup;
+    Tags.of(playAppAsg).add("gu:riffraff:new-asg", "true");
+
+    // ---- Alarms ---- //
+
+    new GuAlarm(this, "NoHealthyInstancesAlarm", {
+      app,
+      alarmName: `[CDK] No healthy instances for ${app} in ${this.stage}`,
```

Comment:
For now I've added `[CDK]` to the start of all of the alarm names. That's because these need to be different from the ones that are defined in the existing cloudformation template. After phase 3 (deleting the existing resources), we should be able to drop the prefix. We did the same thing with [support-frontend](https://github.com/guardian/support-frontend/pull/3639) too